File: ver_py/T02.py
# https://leetcode.cn/problems/jJ0w9p/
class Solution:
    def mySqrt(self, x: int) -> int:
        return int(x**0.5)
if __name__ == "__main__":
    x = int(input())
    print(Solution().mySqrt(x))

File: ver_cpp/T02.cpp
// https://leetcode.cn/problems/jJ0w9p/
#include <bits/stdc++.h>
using namespace std;
class Solution
{
public:
    int mySqrt(int x)
    {
        return (int)sqrt(x);
    }
};

int main()
{
    int x;
    Solution s;
    cin >> x;
    cout << s.mySqrt(x) << endl;
    return 0;
}